appgate/__main__.py

```diff
@@ -168,6 +168,7 @@ def appgate_operator_context(
         spec_directory=Path(spec_directory) if spec_directory else None,
         secrets_key=secrets_key,
         k8s_get_secret=k8s_get_secret,
+        operator_mode=get_operator_name(args.reverse_mode),
```

Comment:
Can we call this `get_operator_mode`? And make it return either a `Literal` or an enum.
Using literals would require less code changes, it would be just changing the return type to e.g.:
```Python
Literal["appgate-operator", "appgate-reverse-operator"]
```

Since we use python3.10 now you could also declare an explicit type alias like this:
```Python
OperatorMode: TypeAlias = Literal["appgate-operator", "appgate-reverse-operator"]
```

See https://docs.python.org/3/library/typing.html#typing.TypeAlias